### PR TITLE
MAINTAINERS: add @schneiderl as org owner

### LINKED ISSUE #5224: Add organization owner: @schneiderl
Hi, @schneiderl! You've been an active tldr-pages organization member for over 6 months (actually 12 months :smile:).
Thanks for sticking around this far and helping out!
According to our [community roles documentation](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md), you're now eligible for becoming an owner in the organization.

That means you will, from now on, be part of the team responsible for performing role changes (like this one!) in the community.
When performing such role transitions, make sure to follow the process described in the [COMMUNITY-ROLES.md](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md) document.

Is that OK with you? Let us know!

Either way, thanks so much for all the work you've done so far. You rock!

### PR DESCRIPTION
**Pull request by schneiderl**
This closes #5224

Once again, thank you for the invite. 😄

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -65,12 +65,12 @@ An automated list can be found [here](https://github.com/orgs/tldr-pages/people)
...
 - **Ein Verne ([@einverne](https://github.com/einverne))**:
   [12 January 2020](https://github.com/tldr-pages/tldr/issues/3738) — present
-- **Lucas Schneider ([@schneiderl](https://github.com/schneiderl))**:
-  [17 January 2020](https://github.com/tldr-pages/tldr/issues/3764) — present
 - **Axel Navarro ([@navarroaxel](https://github.com/navarroaxel))**:
   [05 October 2020](https://github.com/tldr-pages/tldr/issues/4504) — present
 - **bl-ue ([@bl-ue](https://github.com/bl-ue))**:
   [02 February 2021](https://github.com/tldr-pages/tldr/issues/5219) — present
+- Lucas Schneider ([@schneiderl](https://github.com/schneiderl)):
+  [17 January 2020](https://github.com/tldr-pages/tldr/issues/3764) — [8 January 2021](https://github.com/tldr-pages/tldr/issues/5224)
```
Comment (bl-ue) [inline MAINTAINERS.md:73]:
```suggestion
  [17 January 2020](https://github.com/tldr-pages/tldr/issues/3764) — [3 February 2021](https://github.com/tldr-pages/tldr/issues/5224)
```

File: MAINTAINERS.md [hunk 2]
```diff
@@ -97,6 +97,8 @@ An automated list can be found [here](https://github.com/orgs/tldr-pages/people)
   [21 December 2019](https://github.com/tldr-pages/tldr/issues/3672) — present
 - **Zlatan Vasović ([@zlatanvasovic](https://github.com/zlatanvasovic))**:
   [19 June 2020](https://github.com/tldr-pages/tldr/issues/4113) — present
+- **Lucas Schneider ([@schneiderl](https://github.com/schneiderl))**:
+  [4 January 2021](https://github.com/tldr-pages/tldr/issues/5224) — present
```
Comment (bl-ue) [inline MAINTAINERS.md:101]:
```suggestion
  [3 February 2021](https://github.com/tldr-pages/tldr/issues/5224) — present
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Looks like you mostly forgot to change the dates! :smile:

An automated system would sure be nice, as I have difficulty doing this myself ;)

**Review by navarroaxel — APPROVED** [review]
Looks good after the comments of @bl-ue get addressed

**Comment by schneiderl** [comment]
lol I thought I've changed that. thank you for the heads up @bl-ue

**Comment by bl-ue** [comment]
@schneiderl since you're now owner :smile:, will you please tell me, are you able to merge this PR though it's blocked by a request for changes? Just curious :)

**Review by sbrl — APPROVED** [review]
:heart:

**Review by bl-ue — APPROVED** [review]

**Comment by schneiderl** [comment]
@bl-ue I can't tell for sure because it was merged already but I believe I've seen this happen before so I would say yes it is possible.

**Comment by bl-ue** [comment]
Really cool.

**Review by zlatanvasovic — COMMENTED** [review]
Approved.

**Pull request merged**